From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Tue, 10 Apr 2012 17:33:51 +0100
Subject: distclean is very clean

---
 src/makefile.u |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/makefile.u b/src/makefile.u
index a9ef4ed..575ebc7 100644
--- a/src/makefile.u
+++ b/src/makefile.u
@@ -92,6 +92,8 @@ clean:
 veryclean: clean
 	rm -f xsum
 
+distclean: veryclean
+
 b = Notice README cds.c data.c defines.h defs.h equiv.c error.c \
 	exec.c expr.c f2c.1 f2c.1t f2c.h format.c format.h formatdata.c \
 	ftypes.h gram.c gram.dcl gram.exec gram.expr gram.head gram.io \
