From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Wed, 18 Jul 2012 11:15:58 +0200
Subject: sysdep tmpfiles

---
 src/sysdep.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/sysdep.c b/src/sysdep.c
index bca6ff2..01190c3 100644
--- a/src/sysdep.c
+++ b/src/sysdep.c
@@ -98,7 +98,9 @@ Un_link_all(int cdelete)
 	if (!debugflag) {
 		unlink(c_functions);
 		unlink(initfname);
+                unlink(initbname);
 		unlink(p1_file);
+                unlink(p1_bakfile);
 		unlink(sortfname);
 		unlink(blkdfname);
 		if (cdelete && coutput)
@@ -237,6 +239,9 @@ set_tmp_names(Void)
 #endif /*NO_TEMPDIR*/
 		}
 	alloc_names();
+        /* What follows is safe if tmpdir is really
+        a private diectory created by us -- otherwise
+        the file could be a sym link somewhere else....*/
 	sprintf(c_functions, "%s/f2c%ld_func", tmpdir, pid);
 	sprintf(initfname, "%s/f2c%ld_rd", tmpdir, pid);
 	sprintf(blkdfname, "%s/f2c%ld_blkd", tmpdir, pid);
